addons/hr_holidays/views/hr_leave_views.xml

```diff
@@ -353,6 +353,9 @@
             <xpath expr="//group[@name='col_left']" position="attributes">
                 <attribute name="colspan">5</attribute>
             </xpath>
+            <xpath expr="//div[contains(@class, 'o_hr_leave_column')]" position="attributes">
```

Comment:
This gives error - `Error-prone use of @class in view hr.leave.view.form.dashboard.new.time.off (): use the hasclass(*classes) function to filter elements by their classes`

But I don't know a better way to do it :thinking:

Reply:
```suggestion
            <xpath expr="//div[hasclass('o_hr_leave_column')]" position="attributes">
```

Reply:
:innocent: